concordium-sdk/src/main/java/com/concordium/sdk/Client.java

```diff
@@ -138,6 +139,30 @@ public TransactionStatus getTransactionStatus(Hash transactionHash) throws Trans
         return status;
     }
 
+    /**
+     * Get the status of a transaction in a given block.
+     *
+     * @param transactionHash Transaction {@link Hash}
+     * @param blockHash Block {@link Hash}
+     * @return Parsed {@link TransactionStatusInBlock}
+     */
+    public TransactionStatusInBlock getTransactionStatusInBlock(
+            Hash transactionHash,
+            Hash blockHash) throws TransactionNotFoundException {
+        val transactionStatus = server()
+                .getTransactionStatusInBlock(ConcordiumP2PRpc.GetTransactionStatusInBlockRequest.newBuilder()
+                        .setBlockHash(blockHash.asHex())
+                        .setTransactionHash(transactionHash.asHex())
+                        .build());
+        val status = TransactionStatusInBlock.fromJson(transactionStatus);
+
+        if (Objects.isNull(status)) {
+            throw TransactionNotFoundException.from(transactionHash);
```

Comment:
Or was it the block? I think it would be better to throw an exception here containing both the transaction hash and the block hash.  So I suggest creating a new type for this endpoint.

Reply:
done